client/planning-extension/src/extension.ts

```diff
@@ -105,14 +105,40 @@ function onSendBefore(superdesk: ISuperdesk, items: Array<IArticle>, desk: IDesk
 const extension: IExtension = {
     activate: (superdesk: ISuperdesk) => {
         const extensionConfig: IPlanningExtensionConfigurationOptions = superdesk.getExtensionConfig();
-
         const displayTopbarWidget = superdesk.privileges.hasPrivilege('planning_assignments_view')
             && extensionConfig?.assignmentsTopBarWidget === true;
+        const {gettext} = superdesk.localization;
 
         const result: IExtensionActivationResult = {
             contributions: {
                 entities: {
                     article: {
+                        getActions: (item) => [
+                            {
+                                label: gettext('Unlink as Coverage'),
+                                groupId: 'planning-actions',
+                                icon: 'cut',
+                                onTrigger: () => {
+                                    const superdeskArticle = superdesk.entities.article;
+
+                                    if (
```

Comment:
The thing is, we can't add it, because in the angular world permissions work under the hood, so if we do add it it'll still be chopped up. Personally I don't think it's necessary when this is the case, so we can keep it like this

Reply:
add the comment about keeping it up to date